tmail-backend/blob/blobid-list/src/test/java/com/linagora/tmail/blob/blobid/list/BlobStoreConfigurationTest.java

```diff
@@ -166,15 +166,6 @@ void buildingConfigurationShouldThrowWhenDeduplicationPropertieIsOmitted() {
                 "Upgrade note: If you are upgrading from James 3.5 or older, the deduplication was enabled.");
     }
 
-    @Test
-    void singleSaveEnabledShouldDefaultToFalse() {
```

Comment:
Why?

Reply:
I think this is a new feature. So, if the config is omitted, this won't change old logic (backward compatibility)

Reply:
Why do you remove this test?

Reply:
yes, I missed it. I will revert it. No problem here
// Yesterday, when I try to compile the project, my pc is auto shut down. And when it up, Git has been corrupt & error. So, it's hard for me to manual recovery...